src/VisualStudio/IntegrationTest/TestUtilities/AutomationElementExtensions.cs

```diff
@@ -39,7 +39,10 @@ public static IUIAutomationElement FindDescendantByAutomationId(this IUIAutomati
             }
 
             var condition = Helper.Automation.CreatePropertyCondition(AutomationElementIdentifiers.AutomationIdProperty.Id, automationId);
-            var child = parent.FindFirst(TreeScope.TreeScope_Descendants, condition);
+            var child = Helper.Retry(
+                () => parent.FindFirst(TreeScope.TreeScope_Descendants, condition),
+                AutomationRetryDelay,
```

Comment:
It seems like these would be good candidates for default values (or an overload which just takes the lambda).

Reply:
I don't have enough confidence in these specific values to make them the default for all operations. Could be considered in the future if we ever get to a state of high reliability though.